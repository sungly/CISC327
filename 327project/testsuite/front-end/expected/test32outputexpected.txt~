WD 00000000 12345678 50000 ***
WD 00000000 12345678 60000 ***
ES 00000000 00000000 000 ***
