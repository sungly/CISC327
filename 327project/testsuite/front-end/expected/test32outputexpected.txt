WD 12345678 00000000 50000 ***
ES 00000000 00000000 000 ***